tests/e2e/test_gateway_commands.py

```diff
@@ -196,5 +196,12 @@ def test_deploy_account_is_available(protostar: ProtostarFixture):
     )
 
 
+@pytest.mark.usefixtures("init")
+def test_calculate_account_address_is_available(protostar: ProtostarFixture):
+    assert "calculates the account address" in protostar(
+        ["--no-color", "calculate-account-address", "--help"]
+    )
```

Comment:
please write a test which calculates address and outputs something, both in human and JSON forms

Reply:
Added tests